src/main/java/com/limechain/consensus/beefy/BeefyService.java

```diff
@@ -296,6 +297,23 @@ private void finalizeBeefyRound(BigInteger blockNumber) {
         removeFinishedSessions();
     }
 
+    public void reportDoubleVoting(DoubleVotingProof doubleVotingProof) {
+
+        BlockState blockState = stateManager.getBlockState();
+        Runtime runtime = blockState.getRuntime(blockState.getHighestFinalizedHash());
+        runtime.generateBeefyKeyOwnershipProof(doubleVotingProof.getFirst().getCommitment().getAuthoritySetId(),
+                        doubleVotingProof.getFirst().getAuthorityId())
+                .ifPresentOrElse(
+                        key -> runtime.submitReportBeefyDoubleVotingUnsignedExtrinsic(
+                                doubleVotingProof, key.getProof()
+                        ),
```

Comment:
```suggestion
                .ifPresentOrElse(key -> 
                        runtime.submitReportBeefyDoubleVotingUnsignedExtrinsic(
                                doubleVotingProof, key.getProof()
                        ),
```

Let's keep the lambda params and arrow on the first row if we have more than one.